drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageReportDonutPresenter.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.scenariosimulation.client.rightpanel;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import elemental2.dom.HTMLDivElement;
+import org.dashbuilder.dataset.DataSetFactory;
+import org.dashbuilder.displayer.DisplayerSettingsFactory;
+import org.dashbuilder.displayer.client.Displayer;
+import org.dashbuilder.displayer.client.DisplayerCoordinator;
+import org.dashbuilder.displayer.client.DisplayerLocator;
+import org.drools.workbench.screens.scenariosimulation.client.resources.i18n.ScenarioSimulationEditorConstants;
+import org.jboss.errai.common.client.dom.elemental2.Elemental2DomUtil;
+
+@Dependent
+public class CoverageReportDonutPresenter {
```

Comment:
Thanks @Rikkola for `TestResultDonutPresenter`. This class (and the test one) is a sort of clone :P